packages/client/src/validator/CustomEmailValidator_client.ts

```diff
@@ -0,0 +1,17 @@
+import { object, string, array } from 'yup';
+
+export async function validate(msg: object) {
+  const emailSchema = object({
+    from: string().required().email(),
+    to: string().required().email(),
+    subject: string().required(),
+    message: string().required(),
+    replyTo: string().email(),
+    cc: array(string().email()),
+    bcc: array(string().email())
+  });
+  const formattedEmail = emailSchema.cast(msg);
+  const res = await emailSchema.validate(formattedEmail);
+  console.log(res);
```

Comment:
What happens if `res` returns some kind of error?

Reply:
The line 14 (`const res = await emailSchema.validate(formattedEmail)`) would stop emails and return an error message like "ValidationError: to must be a valid email"

The line 15 `(console.log("You have just sent: ", res);` would show a message you've just sent if validation goes successful.

Reply:
`await emailSchema.validate(formattedEmail);` itself returns an error message if the message is not compliant with the format and then it stops there, so users won't see formattedEmail.